composer.json

```diff
@@ -19,13 +19,13 @@
     "require": {
         "php": ">=5.4",
         "behat/behat": "~3.0",
-        "guzzlehttp/guzzle": "4 - 5",
-        "phpunit/phpunit": "~4.0"
+        "guzzlehttp/guzzle": "4 - 5"
     },
 
     "require-dev": {
         "symfony/process": "~2.1",
-        "silex/silex": "~1"
+        "silex/silex": "~1",
+        "phpunit/phpunit": "~4.0"
```

Comment:
I haven't removed assertions from `WebApiExtension/features/bootstrap/FeatureContext.php`, though they would be simple to replace.